packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -33,7 +35,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: '#ffffff',
   },
 
-  sendLabel: {
+  sendLabelRoot: {
     color: theme.palette.text.primary,
     fontFamily: '"Muli SemiBold", sans-serif',
```

Comment:
Also, you can remove the Muli stuff here, should be injected automatically from the MedulasThemeProvider.